GUI/SearchTaskWindow.xaml

```diff
@@ -307,6 +307,13 @@
                                                     </TextBlock>
                                                 </ComboBox.ToolTip>
                                             </ComboBox>
+                                            <CheckBox x:Name="checkBoxClassicSemiSpecific" Content="Semi-Specific Digestion" Margin="6" DockPanel.Dock="Top" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="500">
```

Comment:
what is this for? we already have a semi-specific search button?

Reply:
We have a semi-specific search mode button, but this is a semi-specific protease button. The idea being that if you click this, trypsin turns into semi-trypsin. As such, I've removed semi-trypsin from the protease list. Using a button was cleaner to me than adding a bunch of semi-proteases for comparison (semi-GluC, semi-AspN, etc). I wanted to leave the classic semi-specific search mode intact to compare with for the paper. We can phase it out once published.